src/System.Slices/tools/ILSub/ILSub/ILSub.csproj

```diff
@@ -43,11 +43,13 @@
   </ItemGroup>
   <ItemGroup>
     <Compile Include="Program.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
```

Comment:
Oh.. the file was absent and I removed to compile to tool. I planned to return the ref back and forgot about that.